app/controllers/folders_controller.rb

```diff
@@ -1,7 +1,10 @@
 class FoldersController < ApplicationController
 
   before_action :authenticate_user!
-  
+
+  add_breadcrumb "home", :root_path
+  add_breadcrumb "folders", :folders_path
```

Comment:
Do we need this code? It looks like this functionality is never used in views. Am I wrong?